modules/rest-api/pages/upload-retrieve-root-cert.adoc

```diff
@@ -21,15 +26,18 @@ GET /pools/default
 [#description]
 == Description
 
-The `POST` method and `/controller/uploadClusterCA` URI upload an appropriately configured certificate to the cluster, so that it becomes the root certificate for the cluster.
+The `POST` method and `/controller/uploadClusterCA` URI (which are deprecated in 7.1) upload an appropriately configured certificate to the cluster, so that it becomes the root certificate for the cluster.
 Either the Full Admin or the Security Admin role is required.
 The certificate must be specified as binary data, in a location on the filesystem, for successful upload to occur.
+In versions 7.1 and after, by default, this method and URI can be used on _localhost_ only.
+However, this default setting can be changed, by means of the `POST` method and `/settings/security/allowNonLocalCACertUpload` URI, with the `-d false` flag.
```

Comment:
It's also makes sense to mention POST /settings/security/allowNonLocalCACertUpload` can be called from localhost only (for security reasons)

Reply:
I have added this.